src/any_agent/tools/composio.py

```diff
@@ -0,0 +1,98 @@
+# mypy: disable-error-code="attr-defined,operator,type-arg"
+import inspect as i
+import typing as t
+from types import FunctionType
+
+try:
+    from composio.client.types import Tool
+    from composio.core.provider import AgenticProvider
+    from composio.core.provider.agentic import AgenticProviderExecuteFn
+except ImportError as e:
+    msg = "Composio is not installed. Please install it with `pip install composio`."
+    raise ImportError(msg) from e
+
+
+TYPE_MAPPING = {
+    "string": str,
+    "integer": int,
+    "number": float,
+    "boolean": bool,
+    "array": list,
+    "object": dict,
+}
+
+
+def _get_parameters(tool: Tool) -> list[i.Parameter]:
```

Comment:
Can we use this logic to simplify and clean up the schema -> callable conversion that we do in `tinyagent._load_agent`?